tests/Builder/Query/RandOperatorTest.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MongoDB\Tests\Builder\Query;
+
+use MongoDB\Builder\Expression;
+use MongoDB\Builder\Pipeline;
+use MongoDB\Builder\Query;
+use MongoDB\Builder\Stage;
+use MongoDB\Tests\Builder\PipelineTestCase;
+
+/**
+ * Test $rand query
+ */
+class RandOperatorTest extends PipelineTestCase
+{
+    public function testSelectRandomItemsFromACollection(): void
+    {
+        $pipeline = new Pipeline(
+            Stage::match(
+                Query::expr(
+                    Expression::lt(
+                        0.5,
+                        Expression::rand(),
```

Comment:
Since `RandOperator::ENCODE` is `Encode::Object` and the class has no properties, we can be sure that this encodes as an empty BSON document, correct?

Reply:
Exactly, that's the trick to get an empty object.